### PR TITLE
Added the link to the discord channel

### PR DESCRIPTION
**Pull request by Scramjet911**
Didn't know where I should have added it, so added it before the git commands

### CODE REVIEW

**Comment by rahul799** [comment]
good job @Scramjet911 . I was thinking that it will look better if you make a heading "**Join Us**" just above **how to contribute** section & add the link inside that ?

File: README.md [hunk 1]
```diff
@@ -210,4 +212,4 @@ GitHub provides a Web UI to make communication process easier
 
 - Mentioning Pull Requests or issues using `#(issue_id)`, Read [this](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
 
-- Also, take a look at [this](https://guides.github.com/features/mastering-markdown/), **Markdown or styling** your messages to increase readibilty for PR description and conversations.
\ No newline at end of file
+- Also, take a look at [this](https://guides.github.com/features/mastering-markdown/), **Markdown or styling** your messages to increase readibilty for PR description and conversations.
```
Comment (adiaholic) [inline README.md:215]: @Scramjet911 You have pushed this text a few lines below. I feel like this change is unnecessary. Make sure that your commits are very precise.
Please revert this change and squash.

**Comment by adiaholic** [comment]
@Scramjet911 Never use your master branch to push features. Instead you should make a branch on your fork: eg Scramjet911/some-relevant-name. If you make PRs from the master branch, you won't be able to work on multiple issues simultaneously.

File: README.md (hunk 1 above)
Comment (Scramjet911) [reply README.md:215]: Ok, that was unintentional, I'll fix it

**Comment by adiaholic** [comment]
> @Scramjet911 Never use your master branch to push features. Instead you should make a branch on your fork: eg Scramjet911/some-relevant-name. If you make PRs from the master branch, you won't be able to work on multiple issues simultaneously.

Aah right!

File: README.md (hunk 1 above)
Comment (Scramjet911) [reply README.md:215]: #47 is the new PR. 
That issue came up because I edited the readme.md using github edit. It was fixed when I cloned and added the Link locally.

Comment (Scramjet911) [reply README.md:215]: I also created a branch for the issue this time.
